pkg/autoscaler/autoscaler/calculate/cpu.go

```diff
@@ -65,6 +66,24 @@ func CalculateRecomendedReplicasByCpuCosts(tac *v1alpha1.TidbClusterAutoScaler,
 	if err != nil {
 		return -1, err
 	}
+
+	switch memberType {
+	case v1alpha1.TiDBMemberType:
+		tac.Status.TiDBStatus.Name = string(MetricTypeCPU)
+		tac.Status.TiDBStatus.CurrentValue = cpuSecsTotal
+		tac.Status.TiDBStatus.TargetValue = expectedCpuSecsTotal
```

Comment:
I think the target Value here should be the average Utilization set by user in autoscaler spec, and the currentValue should be the relevant cpu usage ratio.

Reply:
Accept